raster-foundry/raster-foundry
# Align backsplash-server docker and jar with ci

## Overview

This PR aligns the backsplash-server docker container with existing CI processes so that the service can actually start.

### Checklist


### Demo

Currently in staging

### Notes

We discussed whether it makes more sense to switch back to `./sbt` like the other services or keep `sbt` because the helper script is a pain to think about, and decided that if that wasn't the issue we should keep `sbt` and open an issue to remove the helper script elsewhere. That's opened as #4393.

## Testing Instructions

 * `docker-compose -f docker-compose.test.yml build backsplash`
 * `docker run --rm -it raster-foundry-backsplash:latest`
 * it should start
 * `docker-compose build --no-cache backsplash` afterward to restore the normal dev service
